cmd/argocd/commands/app_test.go

```diff
@@ -1849,6 +1858,168 @@ func testApp(name, project string, labels map[string]string, annotations map[str
 	}
 }
 
+func TestWaitOnApplicationStatus_JSONandYAMLOutput(t *testing.T) {
```

Comment:
Could you please also cover scenario when no output flag is provided.

Reply:
have added the test for it,
please check when you are free